src/corelibs/U2View/src/ov_sequence/PanViewRows.cpp

```diff
@@ -74,6 +76,12 @@ bool PVRowData::fitToRow(const QVector<U2Region>& location) {
     return true;
 }
 
+bool PVRowData::containsAnnotationWithName(const QString& name) const {
```

Comment:
You do not need this method at all, because row->key is the name of the annotations inside of the row

Reply:
done